src/doom/am_map.c

```diff
@@ -1273,16 +1240,15 @@ AM_drawFline_Vanilla
 // [crispy] Use DrawWuLine from Heretic
 void AM_drawFline_Smooth(fline_t* fl, int color)
 {
-    byte aaidx = antialias_LUT[color];
-
-    if ((aaidx > 0) && (aaidx <= NUMALIAS))
+    // Use the COLORMAP to produce a range of brightness for the given color
+#define RESHADE(I) (colormaps[I * 256 + color])
```

Comment:
I haven't measured it but might be slightly friendlier to the CPU's prefetcher etc. if these strided reads from `colormaps` were done once in e.g. `AM_initVariables` so this function has to do even less work.

Reply:
Indeed. Could you please implement it this way and initialize an array with pointers to the required colormap fields?